deploy/template.yaml

```
@@ -297,6 +297,8 @@ Resources:
               MOBILE_BACKEND_URL: "/perfTest/mobile/backendUrl"
               MOBILE_FRONTEND_URL: "/perfTest/mobile/frontendUrl"
               MOBILE_TEST_CLIENT_EXECUTE_URL: "/perfTest/mobile/testClientUrl"
+              BTM_AWS_SQS_QUEUE: "/perfTest/txmxconsumeroutputqueuebilling"
+              TXMA_AWS_SQS_QUEUE: "/perfTest/txmxscaledauditeventsqueue"
```

```suggestion
              DATA_BTM_SQS: "/perfTest/data/btm/consumerOutputSQS"
              DATA_TXMA_SQS: "/perfTest/data/txma/auditSQS"
```
Please update the names of these so we are following the format `<POD>_<TEAM>_<VAR_NAME>` or `<POD>_<VAR_NAME>` for the environment variables; and `/perfTest/<pod>/<team>/<varName>` or `/perfTest/<pod>/<varName>` for the parameter store paths.

And then move them so the list remains ordered alphabetically

---

Done